lib/commands/release.js

```diff
@@ -361,7 +360,35 @@ var ReleaseCommand = Command.extend({
 
     return this._repo = new GitRepo(this.project.root);
   },
-  _parsedConfig: function(){
+
+  // HACK ALERT! This hijacks the `Command#registerOptions` which gets called
```

Comment:
You tried doing this in `init` right (I am almost certain you did, but wanted to confirm)?